src/main/java/org/jabref/gui/preferences/websearch/WebSearchTabViewModel.java

```diff
@@ -69,6 +69,9 @@ public class WebSearchTabViewModel implements PreferenceTabViewModel {
     private final FilePreferences filePreferences;
     private final ImportFormatPreferences importFormatPreferences;
 
+    // Added temporary property to store changes made in the UI
+    private final ListProperty<FetcherApiKey> tempApiKeys = new SimpleListProperty<>(FXCollections.observableArrayList());
```

Comment:
I wouldn't call it tempSthg because the lifecycle of this variable is the same as the whole dialog or tab. Please reword it.